### PR TITLE
fix: missing server_metadata

### PR DESCRIPTION
**Pull request by Bento007**
## Reason for Change
rolling back to previously working version of authlib. 
Previously it was upgraded here https://github.com/chanzuckerberg/single-cell-data-portal/commit/a12556f997d9caa1cfbd52453446c5c4af460df3m which caused 502 on the frontend.

## Changes

- add server_metadata to oath config
- roll back to previous authlib version.

## Testing steps
- verify you can login on frontend
- all automated tests pass.

### CODE REVIEW

**Review by nayib-jose-gloria — COMMENTED** [review]
If we're rolling back the authlib change, why do we still need to add in server metadata when its working without it?

**Comment by Bento007** [comment]
it will be needed in the future anyways according to auth0 documentation.

**Comment by nayib-jose-gloria** [comment]
> it will be needed in the future anyways according to auth0 documentation.

I'm still getting a 502 gateway when i try to log in

File: backend/portal/api/app/v1/authentication.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def get_oauth_client(config: CorporaAuthConfig) -> FlaskRemoteApp:
         authorize_url=config.api_authorize_url,
         client_kwargs={"scope": "openid profile email offline_access write:collections"},
         authorize_params={"audience": config.api_audience},
+        server_metadata_url=config.server_metadata_url,
```
Comment (Bento007) [inline backend/portal/api/app/v1/authentication.py:53]:
```suggestion
```

**Comment by Bento007** [comment]
Still looking for where that 502 is coming from.
- rdev does not use cloudfront 
- I did not find any 502 in the logs for the contianer dataportal-rdev-happy-proxy

**Pull request closed without merging**